package-lock.json

```diff
@@ -49,7 +49,7 @@
                 "webpack-cli": "^4.6.0"
             },
             "engines": {
-                "vscode": "^1.60.0"
+                "vscode": "^1.63.0"
```

Comment:
Was worried about this but it matches https://github.com/microsoft/vscode-azureappservice/blob/main/package.json#L10 so I think it's ok.